### PR TITLE
Fix localization

### PR DESCRIPTION
**Pull request by bearfriend**
Fix various localization issues

### CODE REVIEW

File: lang/es-es.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	"presentationModeLabel": "Modo presentación",
+	"presentationModeTitle": "Cambiar al modo presentación",
+	"printTitle": "Abrir una nueva pestaña para imprimir",
+	"printLabel": "Abrir una nueva pestaña para imprimir",
+	"zoomInLabel": "Aumentar",
+	"zoomInTitle": "Aumentar",
+	"zoomOutLabel": "Reducir",
+	"zoomOutTitle": "Reducir"
+}
+diff --git a/lang/cy.json b/lang/cy.json
```
Comment (bearfriend) [inline lang/es-es.json:14]: Note to self: Don't use the github.dev editor.

**Review by jameswklassen — APPROVED** [review]
This seems good. Were the files manually renamed? Has this been tested?

**Comment by bearfriend** [comment]
Yes, they were manually renamed to match what serge sends from the `%LANG%` variable. _Not_ tested. I seems pretty straightforward, so I think it should be fine, but a quick test on your end would be greatly appreciated.

File: lang/nb.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {
-	"downloadTitle": "Last ned",
 	"downloadLabel": "Last ned",
+	"downloadTitle": "Last ned",
 	"pageOfPages": "Page {pageNumber} / {pagesCount}",
 	"presentationModeLabel": "Presentasjonsmodus",
 	"presentationModeTitle": "Bytt til presentasjonsmodus",
+	"printLabel": "Open new tab for printing",
+	"printTitle": "Open new tab for printing",
```
Comment (jadasc) [inline lang/nb.json:8]: Looks like these might have been missed

File: lang/zh-cn.json [hunk 3]
```diff
@@ -1,9 +1,11 @@
 {
-	"downloadTitle": "下载",
 	"downloadLabel": "下载",
+	"downloadTitle": "下载",
 	"pageOfPages": "Page {pageNumber} / {pagesCount}",
```
Comment (jadasc) [inline lang/zh-cn.json:4]: "Page" might need to be translated

Comment (jadasc) [reply lang/zh-cn.json:4]: This might be that case for other languages too

**Review by jadasc — APPROVED** [review]
These changes look good, thanks for making them.

We can merge this PR and submit translation requests after 👍

File: lang/zh-cn.json (hunk 3 above)
Comment (bearfriend) [reply lang/zh-cn.json:4]: Yeah, lots of missing translations for that term.

File: lang/nb.json (hunk 2 above)
Comment (bearfriend) [reply lang/nb.json:8]: I'm not sure that this language is even supported anywhere anymore. Might be able to just remove the file, but it doesn't hurt anything to leave it for now.

**Pull request merged**